### DISCUSSION TITLE
filtering low quality cells after scATAC are integrated

### DISCUSSION THREAD

**Discussion by pengxin2019** [stuart-lab/signac#785] [Q&A] [answered]
Hi Tim,
I have 4 scATAC datasets and I have integrated them following this pipeline https://satijalab.org/signac/articles/integrate_atac.html and the only difference is that I used the reduced peaks created by `reduce` the peaks from 2 of the 4 datasets rather using peaks from 1 dataset.

However, I realized that  when it comes to filtering the low quality cells  I can only use the `nCount_peaks` like below:
`#1
atac.6798.1x <- subset(atac.6798.1x, nCount_peaks > 2000 & nCount_peaks < 30000)
`
I can not filter low quality cells based on `peak_region_fragments` (this parameter does not exist anymore in the new created Seurat object) , `pct_reads_in_peaks` or `nucleosome_signal` like what you demonstrated here https://satijalab.org/signac/articles/pbmc_vignette.html like below:

 ```
#2
pbmc.atac.6798.1x <- subset(
  x = pbmc.atac.6798.1x,
  subset = peak_region_fragments > 3000 &
     peak_region_fragments < 20000 &
     pct_reads_in_peaks > 15 &
    nucleosome_signal < 4 &
   TSS.enrichment > 2
 ) 
```
 `#2` to filter the low quality cells is more strict than the `#1` and thus  I feel like `#2` is doing a better job on filtering than `#1`.  Do you have any suggestions on how to set up a more strict creteria to filter low quality cells  when it comes to scATAC integration?

Thanks
Best
Penny

**Comment by timoast** [comment, marked as answer]
We demonstrate how to compute the nucleosome signal and TSS enrichment metrics in the Signac vignettes. See the end of the [PBMC vignette](https://satijalab.org/signac/articles/pbmc_vignette.html) for an example showing how to compute the percent fragments in blacklist regions metric. Typically, looking at the total counts for the cell along with the nucleosome signal and TSS enrichment score is sufficient to remove low quality cells.
